snap/snapenv/snapenv.go

```diff
@@ -78,6 +78,11 @@ func snapEnv(info *snap.Info) map[string]string {
 // somewhere more reasonable like the snappy module.
 func basicEnv(info *snap.Info) map[string]string {
 	return map[string]string{
+		// NOTE: This uses CoreSnapMountDir because the computed environment
```

Comment:
Nitpick for the style: s/NOTE://

This is a comment. The only reason it exists is to note something. :)